cmd/admin-bucket-handlers.go

```diff
@@ -129,16 +129,17 @@ func (a adminAPIHandlers) SetRemoteTargetHandler(w http.ResponseWriter, r *http.
 	vars := mux.Vars(r)
 	bucket := vars["bucket"]
 
+	if !globalIsErasure {
```

Comment:
Nit: Not sure why access denied is after is the operation possible? If the req does not have admin priv, that should be the first failure.

Reply:
we do not want user thinking it is a admin privilege issue if the gateway/fs mode does not have transition support.